.github/workflows/run-tests.yml

```diff
@@ -74,6 +74,11 @@ jobs:
       single_test_args: ${{ steps.generate_output.outputs.single_test_args }}
       runs_on: ${{ steps.generate_output.outputs.runs_on }}
       dbs: ${{ inputs.test_dbs }}
+      modified_unit_test_suites: ${{ steps.collect_modified_tests.outputs.MODIFIED_UNIT_TEST_SUITES }}
+      modified_integration_test_suites: ${{ steps.collect_modified_tests.outputs.MODIFIED_INTEGRATION_TEST_SUITES }}
+      modified_functional_test_suites: ${{ steps.collect_modified_tests.outputs.MODIFIED_FUNCTIONAL_TEST_SUITES }}
+      modified_functional_test_ndc_suites: ${{ steps.collect_modified_tests.outputs.MODIFIED_FUNCTIONAL_TEST_NDC_SUITES }}
+      modified_functional_test_xdc_suites: ${{ steps.collect_modified_tests.outputs.MODIFIED_FUNCTIONAL_TEST_XDC_SUITES }}
```

Comment:
nit: the rest of the outputs are in lower snake case, can you keep the same format here too?